### PR TITLE
Update app builder build for security scans

### PR DESCRIPTION
**Pull request by joewhite101**
I've run this through the quickstart and everything deploys as expected. 

A test image built with this docker file:
jwhite101/app-builder:yum
https://hub.docker.com/layers/jwhite101/app-builder/yum/images/sha256-e9d3cadfb86496eefba6aab48cee71ac0ef2ab5823901c72489ae1b9489de2ba?context=repo

```
/projects/entando/app-builder$ snyk container test app-builder:yum

Testing app-builder:yum...

Organization:      joewhite101
Package manager:   rpm
Project name:      docker-image|app-builder
Docker image:      app-builder:yum
Platform:          linux/amd64
Licenses:          enabled

✓ Tested 271 dependencies for known issues, no vulnerable paths found.

```

### CODE REVIEW

**Comment by entando-jx** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by:
To fully approve this pull request, please assign additional approvers.
We suggest the following additional approver: **fedelocci**

If they are not already assigned, you can assign the PR to them by writing `/assign @fedelocci` in a comment when ready.

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=entando%2Fapp-builder).

The pull request process is described [here](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process)

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/entando/app-builder/blob/master/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Review by ichalagashvili — APPROVED** [review]

**Comment by entando-jx** [comment]
[APPROVALNOTIFIER] This PR is **APPROVED**

This pull-request has been approved by: *<a href="https://github.com/entando/app-builder/pull/1028#pullrequestreview-627691946" title="Approved">ichalagashvili</a>*

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=entando%2Fapp-builder).

The pull request process is described [here](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process)

<details >
Needs approval from an approver in each of these files:

- ~~[OWNERS](https://github.com/entando/app-builder/blob/master/OWNERS)~~ [ichalagashvili]

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Pull request merged**
